nucypher/cli/commands/alice.py

```diff
@@ -509,7 +509,7 @@ def grant(general_config,
         if not force:
             use_default = click.confirm(f"Confirm default rate {rate}?", default=True)
             if not use_default:
-                rate = click.prompt('Enter rate per period in Wei', type=WEI)
+                rate = click.prompt('Enter rate per period in gwei', type=GWEI)
```

Comment:
We still have to convert it to wei internally. Gwei is just for user facing experience.

Reply:
Ah, yes of course - thanks!

Reply:
:heavy_check_mark: